pytest_fixtures/component/katello_certs_check.py

```diff
@@ -0,0 +1,66 @@
+# katello_certs_check Fixtures
+from pathlib import Path
+
+import pytest
+from broker import Broker
+from fauxfactory import gen_string
+
+from robottelo.constants import CERT_DATA as cert_data
+from robottelo.hosts import Capsule
+from robottelo.hosts import ContentHost
+
+
+@pytest.fixture
+def vm_setup(request):
```

Comment:
Would suggest using more specific fixture names 
```suggestion
def certs_vm_setup(request):
```